tests/unit/BusinessNameForeign.spec.ts

```diff
@@ -17,8 +17,8 @@ describe('BusinessNameForeign', () => {
       })
     const vm: any = wrapper.vm
 
-    expect(vm.businessName).toEqual('')
-    expect(wrapper.emitted('update:businessName').pop()[0]).toBe('')
+    expect(vm.businessName).toEqual(undefined)
+    expect(wrapper.emitted('update:businessName').pop()[0]).toBe(undefined)
```

Comment:
I think there is a `toBeUndefined()` that you can use. Either way.

Reply:
What about line 20? 🙃

Reply:
Oh shoot. I did a global search for toBe(undefined) 😟 Sorry about that! Fixing now!